Missionframework/scripts/server/init_server.sqf

```diff
@@ -126,3 +126,26 @@ execVM "scripts\server\offloading\group_diag.sqf";
 if (KP_liberation_restart > 0) then {
     execVM "scripts\server\game\server_restart.sqf";
 };
+
+if (KP_liberation_limited_zeus) then {
+    zm1 setVariable ["Addons", 0, true];
```

Comment:
Nice, didn't know that this is just stored as `Addons` variable. Guess looked it up in the module functions?
Great finding. 👍

Reply:
Yeah it's from `moduleCurator`, also the same way is used in `virtual` module of 0.97